src/client/datascience/constants.ts

```diff
@@ -659,10 +659,10 @@ export namespace CodeSnippets {
     ];
     export const ChangeDirectoryCommentIdentifier = '# ms-toolsai.jupyter added'; // Not translated so can compare.
     export const ImportIPython = '{0}\nfrom IPython import get_ipython\n\n{1}';
-    export const MatplotLibInitSvg = `import matplotlib\n%matplotlib inline\n${Identifiers.MatplotLibDefaultParams} = dict(matplotlib.rcParams)\n%config InlineBackend.figure_formats = {'svg', 'png'}`;
-    export const MatplotLibInitPng = `import matplotlib\n%matplotlib inline\n${Identifiers.MatplotLibDefaultParams} = dict(matplotlib.rcParams)\n%config InlineBackend.figure_formats = {'png'}`;
-    export const ConfigSvg = `%config InlineBackend.figure_formats = {'svg', 'png'}`;
-    export const ConfigPng = `%config InlineBackend.figure_formats = {'png'}`;
+    export const MatplotLibInitSvg = `import matplotlib\n%matplotlib inline\n${Identifiers.MatplotLibDefaultParams} = dict(matplotlib.rcParams)\n%config InlineBackend.figure_formats = ['svg', 'png']`;
+    export const MatplotLibInitPng = `import matplotlib\n%matplotlib inline\n${Identifiers.MatplotLibDefaultParams} = dict(matplotlib.rcParams)\n%config InlineBackend.figure_formats = ['png']`;
+    export const ConfigSvg = `%config InlineBackend.figure_formats = ['svg', 'png']`;
+    export const ConfigPng = `%config InlineBackend.figure_formats = ['png']`;
```

Comment:
Looks like this has been broken for a long time / forever? For multiples it seems to take {} or [], but for the singular it can't:
![image](https://user-images.githubusercontent.com/812783/129459679-a6ee39b8-d30c-4089-a5bc-12dfba472e78.png)
Docs are a little funny on this, they say set, but the example uses a list?
![image](https://user-images.githubusercontent.com/812783/129459692-32c27d59-3d47-4b6e-b523-812271f062e2.png)

Guessing we didn't notice this before as png is Jupyter default I'm pretty sure. So it would fail to run, but then just get PNG anyways from the Jupyter default.